CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [Unreleased]
+
+### Backward Compatibility Breaking Changes
+
+- Dropped support for PHP 7.0
+
```

Comment:
Looking good - could you also add a section 

```
### Added

- Support for PHP 8.1
```

similar to what's already there in the previous version?

Reply:
Done!